packages/fgtclb/academic-jobs/ext_tables.sql

```diff
@@ -14,13 +14,8 @@ CREATE TABLE tx_academicjobs_domain_model_job (
 	link varchar(255) NOT NULL DEFAULT '',
 	slug varchar(255) NOT NULL DEFAULT '',
 	type int(11) DEFAULT '0' NOT NULL,
-	contact int(11) unsigned NOT NULL DEFAULT '0'
```

Comment:
Why do we remove a relation contact record in favour of a simple direct model fields ... reusing contacts is not that bad - or do I miss something ?

In any case, I would consider this as "heavy" breaking - special as there is no "upgrade" strategy for this (upgrade wizard?). At least a breaking change in the UPGRADE.md is missing.